lib/v2/qbitai/radar.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    'qbitai.com': {
+        _name: '量子位',
+        '.': [
+            {
+                title: '分类-量子位',
```

Comment:
```suggestion
                title: '分类',
```
https://docs.rsshub.app/joinus/new-radar#title

> Do not repeat the website name `_name`